### PR TITLE
Back button 2

### PR DESCRIPTION
**Pull request by ctwoolsey**

### CODE REVIEW

File: app/scripts/services/Session.svc.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    }
+    latestCitations = citations;
+    return citations;
+  }
+
+  function getLastSelectedCitation(){
+    return lastSelectedCitation;
+  }
+
+  function storeSelectedCitation(citation){
```
Comment (abobwhite) [inline app/scripts/services/Session.svc.js]: You remove the return statement from this

File: app/scripts/services/Session.svc.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+angular.module('yourStlCourts').factory('Session', function () {
+  var latestCitations = null;
+  var lastSelectedCitation = null;
+  function getLatestCitations(citations) {
```
Comment (abobwhite) [inline app/scripts/services/Session.svc.js]: Passing citations to a method called `getLatestCitations` seems un-intuitive. Please cleanup logic to explicitly get/set the appropriate citations

File: app/scripts/controllers/citationInfo.ctrl.js [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
   ctrl.paymentData = paymentData;
+  ctrl.selectedCitation = null;
 
   ctrl.selectCitation = function(citation){
-    ctrl.selectedCitation = citation;
-    Courts.findById(citation.court_id).then(function(court){
-      ctrl.selectedCitation.court = court;
-      ctrl.selectedCitation.courtDirectionLink = getCourtDirectionLink(ctrl.selectedCitation);
-    });
+    ctrl.selectedCitation = Session.storeSelectedCitation(citation);
```
Comment (ctwoolsey) [inline app/scripts/controllers/citationInfo.ctrl.js]: done

File: app/scripts/services/Session.svc.js (hunk 1 above)
Comment (ctwoolsey) [inline app/scripts/services/Session.svc.js]: done

File: app/scripts/services/Session.svc.js (hunk 2 above)
Comment (ctwoolsey) [inline app/scripts/services/Session.svc.js]: done

**Comment by ctwoolsey** [comment]
So, initially, we decided to just do it simply as it is done now.  Then I tried tracking states with a back and forward buffer.  It worked well until you got to this scenaria...
home->citationsResult->home(via click on logo)->differentCitationsResult
Now click the back button once on the browser to bring you to the home page.
The next time I click the back button or forward button, how do I know which citation result I should use?  The one from the forward or back buffer?  The problem is that the browser does not tell you if you clicked a back or forward button.

**Pull request merged**